docs/wdc_webengine.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Troubleshoot WDC WebEngine Issues in Tableau 2019.4 and later
+layout: docs
+---
+Prior to Tableau 2019.4, Web Data Connectors used an internal browser in Tableau called Qt WebKit. In Tableau 2019.4 and later, Web Data Connectors use Qt WebEngine (a Chromium-based browser). Qt WebEngine is a significant upgrade over previous browser and WDC developers will benefit from this upgrade with the ability to use modern JavaScript, and Chrome developer tools for debugging WDC code. However,
+some customers might experience problems using their existing connectors in Tableau 2019.4 (and later) because of the change from Qt WebKit to Qt WebEngine in Tableau. This topic describes some of these issues and provides possible solutions.
+
+---
+
+* TOC
+{:toc}
+
+---
+
+### Before you begin
+
+Before diagnosing problems, be sure to launch Tableau Desktop with debugging enabled  (`--remote-debugging-port=9000` option) and check for console messages. These will also be written to the Tableau log file (regular log file on Tableau Desktop; the `tabprotosrv` logs on Tableau Server).
+
+### `tableau.exe -DebugWDC` doesn't work
+
+**Resolution:** Under WebEngine, we use regular Chrome to debug WDCs. Launch Tableau with `--remote-debugging-port=9000` in order to debug it. You no longer need to launch an additional window for debugging.
```

Comment:
i'm not sure what "You no longer need to launch an additional window for debugging." means here

Reply:
@brtal Thanks. Yes. This is your text. Thanks for  that!  I just  wanted to get the ball rolling about the content that needs to go in here. I'll go through it and fix the references to Chrome and Chromium. I'll expand the information we have on the debugging. I am planning to link to the debugging topic where there will be a new section for debugging WebEngine (2019.4 and later).

Reply:
I'm not sure where the "launch a second window" came from --  I think it was referring to the built-in "beta" debugger we used for WebKit.

Reply:
I've revised that section and basically point to the debugging topic where we talk about the Chrome DevTools.